test/onnx/test_op_consistency.py

```diff
@@ -339,6 +342,11 @@ def reason_flaky() -> str:
         reason="ONNX STFT does not support complex results",
         matcher=lambda sample: sample.kwargs.get("return_complex") is True,
     ),
+    dont_care(
+        "unflatten",
+        reason="fixme: Logic not implemented for size 0 inputs in op.Reshape",
```

Comment:
"dont_care" and "fixme" is kinda weird duo ...

Reply:
Yeah we should use fixme instead

Reply:
I would refer to line 180 for the fixme function

Reply:
Updated to fixme